mavros/src/plugins/actuator_control.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * @brief SetpointActuatorControl plugin
+ * @file setpoint_actuator_control.cpp
+ * @author Marcel Stüttgen <stuettgen@fh-aachen.de>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2015 Marcel Stüttgen.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+
+#include <mavros/mavros_plugin.h>
+#include <pluginlib/class_list_macros.h>
+
+#include<std_msgs/Float32MultiArray.h>
+#include<mavros/ActuatorControl.h>
+
+namespace mavplugin {
+
+/**
+ * @brief Setpoint actuator control plugin
+ *
+ * Send setpoint actuator control to FCU controller.
+ */
+class ActuatorControlPlugin : public MavRosPlugin {
+
+ public:
+
+  //constructor
+  ActuatorControlPlugin() : uas(nullptr) { };
+
+  //init function
+  void initialize(UAS &uas_, ros::NodeHandle &nh, diagnostic_updater::Updater &diag_updater) {
+
+    uas = &uas_;
+    sp_nh_ = ros::NodeHandle(nh, "setpoint_actuator_control");
```

Comment:
Lets call this plugin "actuator_control". Direct control, so not exactly setpoints.
Replace sp_nh_ to nh.